site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -41,6 +156,21 @@ spec:
       # label/annotation/player selectors to retrieve an already Allocated GameServer.
       gameServerState: Ready
       # [Stage:Alpha]
+      # [FeatureFlag:CountsAndLists]
+      # priorities:
+      #   counters: # selector for counter current values of a GameServer count
+      #     rooms:
+      #       minCount: 1 # minimum value. Defaults to 0.
+      #       maxCount: 5 # maximum value. Defaults to max(int64)
+      #       minAvailable: 1 # minimum available (current capacity - current count). Defaults to 0.
+      #       maxAvailable: 10 # maximum available (current capacity - current count) Defaults to max(int64)
+      #   lists:
+      #     players:
+      #       containsValue: "x6k8z" # only match GameServers who has this value in the list. Defaults to "", which is all.
+      #       minAvailable: 1 # minimum available (current capacity - current count). Defaults to 0.
+      #       maxAvailable: 10 # maximum available (current capacity - current count) Defaults to 0, which translates to max(int64)
+      #
```

Comment:
```suggestion
      # counters: # (Alpha, CountsAndLists feature flag) Counter actions to perform during allocation.
      #   rooms:
      #     action: increment # Either “Increment” or “Decrement” the Counter’s Count.
      #     amount: 1 # Amount is the amount to increment or decrement the Count. Must be a positive integer.
      #     capacity: 5 # Amount to update the maximum capacity of the Counter to this number. Min 0, Max int64.
      # lists: # (Alpha, CountsAndLists feature flag) List actions to perform during allocation.
      #   players:
      #     addValues: # appends values to a List’s Values array. Any duplicate values will be ignored
      #       - x7un
      #       - 8inz
      #     capacity: 40 # Updates the maximum capacity of the Counter to this number. Min 0, Max 1000.
      # 
```

Please add this in (this was the actions mentioned previously) - and also make sure it shows up in the field description section below.

@igooch did I mess anything up there?